commands/apps/dashboard.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+let co   = require('co');
+let cli  = require('heroku-cli-util');
+let time = require('../../lib/time');
+let _    = require('lodash');
+
+const NOW       = new Date().toISOString();
+const YESTERDAY = new Date(new Date().getTime() - (24*60*60*1000)).toISOString();
+
+let empty = o => Object.keys(o).length === 0;
+
+function displayFormation (formation) {
+  formation = _.groupBy(formation, 'size');
+  formation = _.map(formation, (p, size) => `${cli.color.yellow(_.sumBy(p, 'quantity'))}:${cli.color.cyan(size)}`);
+  cli.log(`    Dynos: ${formation.join(' ')}`);
+}
+
+function displayErrors (metrics) {
+  let errors = [];
+  if (metrics.routerErrors) {
+    errors = errors.concat(_.toPairs(metrics.routerErrors.data).map(e => cli.color.red(`${_.sum(e[1])} ${e[0]}`)));
+  }
+  if (metrics.dynoErrors) {
+    metrics.dynoErrors.filter(d => d).forEach(dynoErrors => {
+      errors = errors.concat(_.toPairs(dynoErrors.data).map(e => cli.color.red(`${_.sum(e[1])} ${e[0]}`)));
+    });
+  }
+  if (errors.length > 0) cli.log(`    ${errors.join(', ')} - see details with ${cli.color.yellow('heroku apps:errors')}`);
+}
+
+function displayMetrics (metrics) {
+  let ms = ''; let rpm = '';
+  if (metrics.routerLatency && !empty(metrics.routerLatency.data)) {
+    ms = _.pad(`${_.round(_.mean(metrics.routerLatency.data.latency_p50))} ms`, 6);
+  }
+  if (metrics.routerStatus && !empty(metrics.routerStatus.data)) {
+    rpm = metrics.routerStatus  ? `${_.round(_.sum(_.flatten(_.values(metrics.routerStatus.data)))/24/60)} rpm` : '';
+  }
+  cli.log(`    ${cli.color.green(ms)} ${cli.color.yellow(rpm)}`);
+}
+
+function* run (context, heroku) {
+  function favoriteApps () {
+    return heroku.request({
+      host: 'longboard.heroku.com',
```

Comment:
I'd really like to use it for this, any suggestions on what you think the right way to go is?